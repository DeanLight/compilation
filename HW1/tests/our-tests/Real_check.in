1.
.2
0.0
0.
.0
12.31
.002
.23400
1.000101