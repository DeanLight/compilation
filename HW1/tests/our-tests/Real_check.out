1 REAL 1.
2 REAL .2
3 REAL 0.0
4 REAL 0.
5 REAL .0
6 REAL 12.31
7 REAL .002
8 REAL .23400
9 REAL 1.000101
9 EOF 
